infrastructure/code/template/details.html

```diff
@@ -234,14 +234,14 @@ <h2 class="data-flow-header">
                 Data Flow - {{len .DataFlow}} {{if gt (len .DataFlow) 1}}steps{{else}}step{{end}}
               </h2>
               <div class="data-flow-body">
-                {{range $fileName, $dataFlowItems := .DataFlowTable}}
+                {{range $fileName := .DataFlowKeys}}
                 <div class="data-flow-file">
                   {{$.FileIcon}}
                   <span class="data-flow-file-name">{{$fileName}}</span>
                 </div>
                 <table class="data-flow-table">
                   <tbody>
-                    {{range $dataFlowItems}}
+                    {{range index $.DataFlowTable $fileName}}
```

Comment:
the `$fileName` is taken from the `.DataFlowKeys` slice, which is indexed so we know the order, then the DataFlowTable map is accessed by said `$fileName` key